docs/Project.toml

```diff
@@ -1,5 +1,6 @@
 [deps]
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+HOHQMesh = "e4f4c7b8-17cb-445a-93c5-f69190ed6c8c"
```

Comment:
```suggestion
```
Not required/recommended (Makes you maintain a compat entry as well)

Reply:
That makes sense. When I ran the docs build locally it modified the `Project.toml` with the package. Should that not happen?

Reply:
Ah, the `make.jl` has a `using HOHQMesh` in it from the [initial docs setup](https://github.com/trixi-framework/HOHQMesh.jl/blame/main/docs/make.jl#L3). This probably doesn't need to be there right?

Reply:
No, I think the `using` is correct. The fact that `Project.toml` gets modified seems to me like it's the usual way how it is done. I've wondered about it several times, but never really bothered to investigate it in earnest